agent/structs/structs_test.go

```diff
@@ -1588,30 +1588,22 @@ func TestStructs_validateMetaPair(t *testing.T) {
 	}
 }
 
-func TestDCSpecificRequestCacheInfoKey(t *testing.T) {
-	assertCacheInfoKeyIsComplete(t, &DCSpecificRequest{}, nil)
+func TestDCSpecificRequest_CacheInfoKey(t *testing.T) {
+	assertCacheInfoKeyIsComplete(t, &DCSpecificRequest{})
 }
 
-func TestNodeSpecificRequestCacheInfoKey(t *testing.T) {
-	assertCacheInfoKeyIsComplete(t, &NodeSpecificRequest{}, nil)
+func TestNodeSpecificRequest_CacheInfoKey(t *testing.T) {
+	assertCacheInfoKeyIsComplete(t, &NodeSpecificRequest{})
 }
 
-func TestServiceSpecificRequestCacheInfoKey(t *testing.T) {
-	ignoredFields := map[string]bool{
-		// TODO: should this filed be included?
-		"ServiceKind": true,
-	}
-
-	assertCacheInfoKeyIsComplete(t, &ServiceSpecificRequest{}, ignoredFields)
+func TestServiceSpecificRequest_CacheInfoKey(t *testing.T) {
+	// TODO: should ServiceKind filed be included in the key?
```

Comment:
Yes it should be, it can influence the output of the topology endpoint for the UI